### ISSUE TITLE
Release SID-Bench and interruption detection model on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [xkx-hub/SID-bench#1]
Hi @xkx-hub 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2603.24144.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw in your paper that you plan to release the **SID-Bench** benchmark and the associated code. It'd be great to make the checkpoints and the dataset available on the 🤗 hub, to improve their discoverability and visibility within the speech and dialogue community. We can add tags so that people find them when filtering https://huggingface.co/models and https://huggingface.co/datasets.

## Uploading models

See here for a guide: https://huggingface.co/docs/hub/models-uploading.

In this case, since you mention an LLM-based model, you could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverage the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

## Uploading dataset

Would be awesome to make **SID-Bench** available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/sid-bench")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the audio samples and labels in the browser.

Let me know if you're interested or need any help regarding this!

Cheers,

Niels
ML Engineer @ HF 🤗
